tensorflow_addons/layers/polynomial.py

```diff
@@ -40,15 +40,13 @@ class PolynomialCrossing(tf.keras.layers.Layer):
 
     Example:
 
-    ```python
     # after embedding layer in a functional model:
```

Comment:
Your example does not use functional model. So either remove this line or change your example to functional API.